azure-pipelines.yml

```diff
@@ -182,7 +182,13 @@ stages:
               arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
             env:
               Token: $(dn-bot-dnceng-artifact-feeds-rw)
-        - script: eng\common\cibuild.cmd
+        - script: eng\common\Build.ps1
```

Comment:
What's this do differently than the cibuild?

Reply:
Basically it gives us the granularity of control needed to avoid either running the tests twice or build twice (since cibuild.cmd always calls build and test). I have otherwise copied the current behavior of cibuild.cmd.